### PR TITLE
Updates to jBPM website to make it more useful and easy to use

### PR DESCRIPTION
**Pull request by mswiderski**
updated website to make it a bit more appealing and better to find basic information
new page for getting started to know how to easily get going on the first try

screen shot 2018-07-10 at 11 00 35

### CODE REVIEW

File: _config/site.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 encoding: UTF-8
-title: jBPM - Open Source Business Process Management
-description: jBPM is a flexible Business Process Management (BPM) Suite.
-keywords: metaheuristic, heuristic, constraint satisfaction, constraint programming, operations research, NP-complete, NP-hard, optimization, planning, scheduling, java, open source, software, library, solver, tabu search, simulated annealing, pareto
+title: jBPM - Open Source Business Automcation Toolkit
```
Comment (jhrcek) [inline _config/site.yml]: "Automcation" sounds like a sales pitch for tomcat. Is that intentional? :D

File: _config/site.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+description: jBPM is a toolkit for building business application backed by solid business automation capabilities.
```
Comment (jhrcek) [inline _config/site.yml]: application -> applications

File: download/download.adoc [hunk 3]
```diff
@@ -73,6 +56,6 @@ More info at http://blog.athico.com/2015/06/drools-jbpm-get-dockerized.html[this
 
 === Nightly snapshots
 
-Nightly snapshots are unstable binaries, build by link:../code/continuousIntegration.html[our build server].
+Nightly snapshots are unstable binaries, build by our build server.
```
Comment (jhrcek) [inline download/download.adoc]: build -> built (the first occurrence, the 2nd is ok)

File: index.html.haml [hunk 4]
```diff
@@ -1,7 +1,73 @@
...
+  %a.alert-link(href="#{site.pom.latestFinal.releaseNotesVersion}") jBPM #{site.pom.latestFinal.version} has been released. Click here to see what's new.
+
+.row
+  .col-md-8(style="margin-top: -17px;margin-bottom: 20px")
+    :markdown
+      ## What is jBPM?
+
+      **jBPM** is a toolkit for building business applications.
+
+      **Business application is a custom application built with selected framework and capabilities. Capabilities can be (among others) persistence, massaging, transactions, business processes, business rules, planning solutions.**
```
Comment (jhrcek) [inline index.html.haml]: massaging -> messaging. Or does jBPM have features I don't know of? ;)

File: index.html.haml [hunk 5]
```diff
@@ -1,7 +1,73 @@
...
+      jBPM originates from BPM (Business Process Management) but it has evolved to enable users to pick their own path in business automation.
+      It provides various capabilities that simplify and externalize business logic into reusable assets such as cases, processes, decision tables and more.
+    %ul
+      %li business processes (BPMN2)
+      %li case management (BPMN2 and CMMN)
+      %li decision management (DMN)
+      %li business rules (DRL)
+      %li business optimisation (Solver)
+
+    %p jBPM can be used as standalone service or embedded in custom service. It does not mandates any of the frameworks to be used, it can be successfully used in
```
Comment (jhrcek) [inline index.html.haml]: does not mandates -> does not mandate

File: learn/gettingStarted.adoc [hunk 6]
```diff
@@ -0,0 +1,194 @@
+= Getting started
+:awestruct-description: Getting started guide for jBPM. Start quickly and easy.
```
Comment (jhrcek) [inline learn/gettingStarted.adoc]: easy -> easily

File: download/download.adoc [hunk 7]
```diff
@@ -40,25 +41,7 @@ The jBPM binaries include documentation, examples and sources.
...
-----
-    <dependency>
-      <groupId>org.jbpm</groupId>
-      <artifactId>jbpm-test</artifactId>
-      <version>#{site.pom.latestFinal.version}</version>
-    </dependency>
-----
-
-This is similar for Gradle, Ivy and Buildr.
+* Evaluation version of Red Hat Process Automation Manager: Check http://www.jboss.com/downloads/[the product download site].
```
Comment (rsynek) [inline download/download.adoc]: http://www.jboss.com/downloads is redirected to https://access.redhat.com/downloads - is there a reason for not putting the target URL there directly?

Comment (mswiderski) [reply download/download.adoc]: frankly speaking not sure as it was already there, I just changed the BPM Suite to PAM

**Comment by mswiderski** [comment]
@rsynek workbench is our primary tool for users so having it as first thing might be nice catch - it's future rich and can provide a lot of usefulness especially when you starting. Plus getting started use that as entry point too.

**Comment by rsynek** [comment]
@mswiderski ok, if this is the intention it makes sense.

File: learn/gettingStarted.adoc [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+Additional users can be created via jBPM Console Admin section.
+
+'''
+
+*KIE Server Swagger documentation*
+
+KIE Server is the execution server that provides various capabilities
+
+- BRM (business rules)
+- BPM (business processed)
```
Comment (MarianMacik) [inline learn/gettingStarted.adoc]: business **processes**

**Comment by cristianonicolai** [comment]
@mswiderski awesome!

**Pull request merged**